viya_ark_library/k8s/sas_kubectl.py

```diff
@@ -11,7 +11,7 @@
 ####################################################################
 import json
 
-from subprocess import CalledProcessError, check_output, DEVNULL
+from subprocess import CalledProcessError, check_output, STDOUT
```

Comment:
@sasjowood is this ok to change from DEVNULL?

Reply:
It needs to be STDOUT or we won't be able to capture the following error:
Error from server (BadRequest): container "sas-annotations" in pod "sas-annotations-6dd4f6cb5-7ft4f" is waiting to start: PodInitializing
It is not true that we'll get a lot more unexpected messages.

Reply:
@cuddlehub tested the use of DEVNULL in her use-case and found it was insufficient. The use of STDOUT seems necessary to provide the desired functionality.